RadixWallet/Clients/URLFormatterClient/URLFormatterClient+Live.swift

```diff
@@ -23,10 +23,15 @@ extension URLFormatterClient: DependencyKey {
 	#endif
 
 	private static func makeURL(url: URL, imageServiceURL: URL, size: CGSize) -> URL {
-		let originItem = URLQueryItem(name: "imageOrigin", value: url.absoluteString)
-		let sizeItem = URLQueryItem(name: "imageSize", value: imageSizeString(size: size))
-
-		return imageServiceURL.appending(queryItems: [originItem, sizeItem])
+		if url.isDataURL {
+			let imageOrigin = url.absoluteString.addingPercentEncoding(withAllowedCharacters: .alphanumerics) ?? ""
+			let imageSize = imageSizeString(size: size)
+			return URL(string: "\(imageServiceURL.absoluteString)/?imageOrigin=\(imageOrigin)&imageSize=\(imageSize)")!
```

Comment:
Why not use `URL.appending(queryItems:)` instead of manually buliding it and force casting it?

Reply:
oh I see. Then I would add a comment similar to this on the code so we avoid changing it in the future. Also, may be useful adding some unit tests?